apps/web/app/(use-page-wrapper)/settings/(settings-layout)/layout.tsx

```diff
@@ -1,24 +1,47 @@
+import { unstable_cache } from "next/cache";
 import { cookies, headers } from "next/headers";
 import { redirect } from "next/navigation";
 import React from "react";
 
 import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
+import type { AppFlags } from "@calcom/features/flags/config";
+import { FeaturesRepository } from "@calcom/features/flags/features.repository";
 
 import { buildLegacyRequest } from "@lib/buildLegacyCtx";
 
 import type { SettingsLayoutProps } from "./SettingsLayoutAppDirClient";
 import SettingsLayoutAppDirClient from "./SettingsLayoutAppDirClient";
 
+const getTeamFeatures = unstable_cache(
+  async (teamId: number) => {
+    const featuresRepository = new FeaturesRepository();
+    return await featuresRepository.getTeamFeatures(teamId);
+  },
+  ["team-features"]
+);
+
```

Comment:
when do we invalidate this cache? 

no need to add `tags` or `revalidate` time?

Reply:
This is hard one because we only ever add this from the DB manually at the moment. I guess we can add an invalidate time here